core/include/drivers/cbmem_console.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Google Limited
+ */
+#ifndef CBMEM_CONSOLE_H
+#define CBMEM_CONSOLE_H
+
+#include <types_ext.h>
+#include <drivers/serial.h>
+
+struct cbmem_console {
```

Comment:
do these 2 struct need to be defined in this global header file?

Reply:
No, moving them to the .c file.